conf/messages

```diff
@@ -138,6 +138,8 @@ inspection_needed.inspection_site.p1=You must attend any inspections in the orde
 inspection_needed.what_next.warning=You may be charged a penalty if you do not arrive for an inspection
 inspection_needed.banner=Inspection required
 
+inspection_needed.defra_transit.heading=For your DEFRA transit inspection
```

Comment:
worth moving this line to `Inspection needed import` section from `Inspection needed common` as this is not common?

Reply:
can move, but i think we should specify import/export specific keys only if there is a difference in the content, 
at the moment i see we have same duplicated content in export/import keys which ideally should be common content without import/export key prefixes